test/test_mutable_binheap.jl

```diff
@@ -230,4 +249,4 @@ end
     end
     
 
-end # @testset MutableBinheap
+end # @testset MutableBinheap
```

Comment:
This was probably your editor removing the last line of the file?  If so, can you set it not do do so, so we don't get extraneous changes like this?  Thanks.